pkgs/build-support/setup-hooks/compress-man-pages.sh

```diff
@@ -18,11 +21,16 @@ compressManPages() {
         fi
     done
 
+    # replace symlinks with uncompressed manpages
     for f in "$dir"/share/man/*/* "$dir"/share/man/*/*/*; do
-        if [ -L "$f" -a -f `readlink -f "$f"`.gz ]; then
-            ln -sf `readlink "$f"`.gz "$f".gz && rm "$f"
```

Comment:
I wonder why the last `&&` is needed are build hooks executed with `set -e` anyway?